bin/init-localstack.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+const { DynamoDB } = require('aws-sdk');
+const DynamoTools = require('./dynamo-tools');
+
+const regions = ['us-west-2'];
+const clients = regions.reduce((acc, region) => {
+  acc[region] = new DynamoDB({
+    accessKeyId: 'fakeKeyId',
+    secretAccessKey: 'fakeSecretAccessKey',
+    endpoint: 'http://localhost:4569',
```

Comment:
Latest version of `localstack` runs all services on port 4566 only.

```suggestion
    endpoint: 'http://localhost:4566',
```

Reply:
Not using the latest version of localstack.